changelog/updates/2021-12-02-linux-update.md

```diff
@@ -0,0 +1 @@
+- Linux ([5.15.6](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tag/?h=v5.15.6))
```

Comment:
The link above is technically correct.
But does it mean that the bot will create the changelog entry for every Kernel release? e.g. one for 5.15.6, another for 5.15.7.
Not sure how we can combine all the entries into a meaningful one.

Reply:
Ideally it would be https://lwn.net/Articles/877286/ but good luck in figuring this out in a github action. The script that processes the changelog files could, for example, merge the entries into something like `Linux ([5.15.5](url), [5.15.6](url))`.

Reply:
Ok, I'll change the kernel github action to do something like this:
```sh
curl -sA "Chrome" -L 'http://www.google.com/search?hl=en&q=site%3Alwn.net+linux+5.15.6' -o search.html
url=$(grep -o 'https://lwn.net/Articles/[0-9]\+' search.html | head -n 1)
if [[ ! "${url}" ]]; then url=https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tag/?h=v5.15.6; fi
```

Reply:
Here's the job on my fork: https://github.com/krnowak/coreos-overlay/runs/4395859702?check_suite_focus=true
Here's the code doing it: https://github.com/krnowak/coreos-overlay/commit/eeb6e1e3057ec4343579a398769167aae8e5b282

Reply:
And the result: https://github.com/krnowak/coreos-overlay/pull/2/files

I'll file a PR for it when I'm back.

Reply:
Here it is, I'll update this PR manually in the meantime.

Reply:
Yh, forgot to paste the link: #1470

Reply:
>  Not sure how we can combine all the entries into a meaningful one.


@dongsupark Good that you bring this topic up, I actually found it limiting that we used to only show the latest release notes and the user has to search for all the point releases between the version used in the previous release and the new one. Now thanks to this automation we will have all point release changelogs easily available. For the release note formatting we could come up with something like that `Linux 5.15.6 (includes 5.15.5, 5.15.4, 5.15.3)` when the previous release used `5.15.2`.

Reply:
Looks like https://github.com/flatcar-linux/coreos-overlay/pull/1489 was created with a correct changelog. Thanks!